lambdas/user-list-search-corpus-indexing/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "tsconfig/lambda.json",
   "compilerOptions": {
     "outDir": "dist",
-    "rootDir": "src"
+    "rootDir": "src",
+    "strictNullChecks": false
```

Comment:
Turned off to not conflict with https://github.com/Pocket/pocket-monorepo/pull/751

Reply:
#751 is merged